airflow/providers/google/cloud/sensors/dataplex.py

```diff
@@ -114,3 +115,76 @@ def poke(self, context: Context) -> bool:
         self.log.info("Current status of the Dataplex task %s => %s", self.dataplex_task_id, task_status)
 
         return task_status == TaskState.ACTIVE
+
+
+class DataplexJobStatusSensor(BaseSensorOperator):
+    """
+    Check the status of the Dataplex data scan job.
```

Comment:
Wouldn't the next phrase be more specific?
```
Check the status of the Dataplex data scan job is SUCCEEDED.
```

Reply:
yes, changed